tools/pipelines/deploy-website.yml

```diff
@@ -49,7 +49,7 @@ variables:
   - name: shouldRetainGuardianAssets
     value: ${{ or(
       eq(parameters.guardianAssetRetentionOverride, 'force'),
-      and(eq(parameters.guardianAssetRetentionOverride, 'default'))
+      eq(parameters.guardianAssetRetentionOverride, 'default')
```

Comment:
When are we supposed to retain these assets? When we actually deploy the site? Should this policy align with the deployment check?